grafana-formula/grafana/init.sls

```diff
@@ -136,6 +154,29 @@ grafana-sap-netweaver-dashboards:
   pkg.removed
 {%- endif %}
 
+{% if use_podman %}
+grafana-container:
+  file.managed:
+    - names:
+      - /etc/containers/systemd/grafana.container:
+        - source: salt://grafana/files/containers/grafana.container
+      - /etc/containers/systemd/grafana.volume:
+        - source: salt://grafana/files/containers/grafana.volume
+    - user: root
+    - group: root
+    - mode: 644
+  module.run:
+    - name: service.systemctl_reload
+  service.running:
+    - name: grafana
+    - enable: true
+    - watch:
+      - file: /etc/containers/systemd/grafana.*
+      - file: /etc/grafana/provisioning/datasources/datasources.yml
```

Comment:
The service state/execution module calls won't work on SLE Micro.

Reply:
`state.apply` is executed inside a transaction (think: `transactional-update run salt-call state.apply ...`) and there is no dbus access inside the transaction. At least not as of today, but enabling it also comes with problems...

We have a card to enable SUMA to avoid the `transactional-update` wrapping, but it's not picked up yet.

Just to be clear, systemd and podman work together. Controlling that with Salt won't work when targeting transactional systems.

Reply:
Thanks. That's a good point.